### ISSUE TITLE
Comment inside Free Form Description not allowed

### ISSUE THREAD

**Issue by jacob-wieland-gematik** [cucumber/gherkin#333]
### 👓 What did you see?

If you write a feature with a comment inside of the description, the file cannot be parsed successfully. (I assume the same is true for all free form descriptions). The parser once entering the comment state will not return to the description state after parsing the comment.

### ✅ What did you expect to see?

From the documentation, a comment does not end a description section. It supposedly is ended only by one of the keywords and comment is not mentioned as one of the keywords.

### 🔬 How could we reproduce it?

```
Feature: X Z Y

Some Description
# Comment
Some more Description
```

**Comment by kieran-ryan** [comment]
Thanks a mill for report @jacob-wieland-gematik, just checking is it the `Java` parser implementation you are using?

**Comment by jacob-wieland-gematik** [comment]
Yes, this is of course true for all the parsers. I have now changed the base grammar so that it should be fixed in all parsers.

**Comment by jacob-wieland-gematik** [comment]
Could you maybe approve re-running of the tests with the latest changes @kieran-ryan ?
